cg/meta/upload/scoutapi.py

```diff
@@ -112,7 +127,11 @@ def generate_config(self, analysis_obj: models.Analysis, rank_score_threshold: i
         else:
             LOG.info("family of 1 sample - skip pedigree graph")
 
-        return data
+        # Validate that the config is correct
+
+        scout_case = ScoutCase(**data)
```

Comment:
could also be done with ScoutCase.parse_obj(data)
Not saying its better but i've always done that instead of unpacking

Reply:
Ok! sounds like it would be similar. I like that it will strip the `None` values when converting back to dict

Reply:
That can be stripped anyway and can be configured when dumping model:
https://pydantic-docs.helpmanual.io/usage/exporting_models/#modeldict